test/utils.ts

```diff
@@ -258,3 +263,75 @@ export function getLocalEip712Signature(
     // @ts-ignore
     return sigUtil.signTypedData_v4(privateKey, { data: typedRequestData });
 }
+
+export async function deployHub(
+    penalizer: string = constants.ZERO_ADDRESS,
+    configOverride: Partial<RelayHubConfiguration> = {}
+): Promise<RelayHubInstance> {
+    const relayHubConfiguration: RelayHubConfiguration = {
+        ...defaultEnvironment.relayHubConfiguration,
+        ...configOverride
+    };
+    return await RelayHub.new(
+        penalizer,
+        relayHubConfiguration.maxWorkerCount,
+        relayHubConfiguration.minimumEntryDepositValue,
+        relayHubConfiguration.minimumUnstakeDelay,
+        relayHubConfiguration.minimumStake
+    );
+}
+
+export async function getGaslessAccount(): Promise<AccountKeypair> {
+    const a = ethWallet.generate();
```

Comment:
don't call constants with a single letter please (unless it's an index or something like that).
Also you don't actually use the wallet itself, so just deconstruct `const {getPrivateKey} = ethWallet.generate()`, so it's more readable.

Reply:
I renamed the function to with a more declarative name, but I don't think we can deconstruct the object, because the function won't be bound to any object. (internally the functions uses [this](https://github.com/ethereumjs/ethereumjs-wallet/blob/master/src/index.ts#L506), so we cannot just deconstruct it. (we could deconstruct it and bind it again, but it doesn't make sense to me)

Reply:
hm good point! rookie mistake that... ;)

Reply:
the const name should change tho...